src/manage_sql_nvts.c

```diff
@@ -1592,6 +1468,10 @@ update_nvts_from_vts (entity_t *get_vts_response,
       nvti_t *nvti = nvti_from_vt (vt);
 
       insert_nvt (nvti);
+// TODO: perhaps use chunked inserts? The next line is simply
```

Comment:
Please rather remove this.  We will not forget about the chunking, and these TODOs
clutter up the code because there are so many possibilities.  We always have the old
code to refer to anyway.

Reply:
Removed.